docs/index.rst

```diff
@@ -8,10 +8,10 @@ Volue Insight API Python library documentation
 
 :Maintainer: <support.insight@volue.com>
 :Organization: `Volue Insight`_
-:Version: |version|
+:Version: |release|
```

Comment:
I changed this from variable `version` to `release` because we dont set version, so it was [showing blank](https://wattsight-volue-insight-timeseries.readthedocs-hosted.com/en/latest/)
So I used the `release` variable instead.